test/CMakeLists.txt

```diff
@@ -15,14 +15,15 @@ add_library(valhalla_test
   ${VALHALLA_SOURCE_DIR}/third_party/microtar/src/microtar.h
   ${VALHALLA_SOURCE_DIR}/third_party/microtar/src/microtar.c
   )
-target_include_directories(valhalla_test PUBLIC
-  ${VALHALLA_SOURCE_DIR}/test
-  ${VALHALLA_SOURCE_DIR}/test/gurka
-  ${VALHALLA_SOURCE_DIR}/third_party/protozero/include
-  ${VALHALLA_SOURCE_DIR}/third_party/just_gtfs/include
-  ${VALHALLA_SOURCE_DIR}/third_party/libosmium/include
-  ${robinhoodhashing_include_dir}
-  ${VALHALLA_SOURCE_DIR}/third_party/microtar/src)
+target_include_directories(valhalla_test
+  PUBLIC
+    ${VALHALLA_SOURCE_DIR}/test
+    ${VALHALLA_SOURCE_DIR}/test/gurka
+  SYSTEM
+    ${VALHALLA_SOURCE_DIR}/third_party/just_gtfs/include
+    ${VALHALLA_SOURCE_DIR}/third_party/protozero/include
+    ${VALHALLA_SOURCE_DIR}/third_party/libosmium/include
+    ${VALHALLA_SOURCE_DIR}/third_party/microtar/src)
```

Comment:
yeah this doesnt work i recently learned this from my other projects. iirc you need to separate them like this:

```
target_include_directories(valhalla_test
  PUBLIC
    ${VALHALLA_SOURCE_DIR}/test
   ${VALHALLA_SOURCE_DIR}/test/gurka)

target_include_directories(valhalla_test
  SYSTEM PRIVATE
    ${VALHALLA_SOURCE_DIR}/third_party/just_gtfs/include
    ${VALHALLA_SOURCE_DIR}/third_party/protozero/include
    ${VALHALLA_SOURCE_DIR}/third_party/libosmium/include
    ${VALHALLA_SOURCE_DIR}/third_party/microtar/src)
```

Reply:
Hmpf, alright, thanks for the hint